tests/functional/builtins/codegen/test_create_functions.py

```diff
@@ -443,11 +436,13 @@ def should_fail(target: address, arg1: String[129], arg2: Bar):
     sig = keccak("should_fail(address,string,(string))".encode()).hex()[:10]
     encoded = abi.encode("(address,string,(string))", (f.address, FOO, BAR)).hex()
     with tx_failed():
-        w3.eth.send_transaction({"to": d.address, "data": f"{sig}{encoded}"})
+        env.execute_code(d.address, env.deployer, f"{sig}{encoded}")
```

Comment:
maybe we should rename `execute_code` to `raw_call`, what do you think?

Reply:
In the evms in this is called `apply_message` (pyevm) and `message_call` (revm).
`execute_code`, `raw_call` or `eth_call`.. Hard to say lol

Reply:
let's call it anything with `call` in it. `execute_code` might make the reader think that you provide code to be executed, but it's actually loading the code at the target address like a message call. i think `message_call` or `raw_call` are both good.